### PR TITLE
New contact-us page

### LINKED ISSUE #792: Add a Contact Us page
A MAAS equivalent to the Juju Contact us page (https://jujucharms.com/docs/stable/contact-us).

### PR DESCRIPTION
**Pull request by degville**
fixes #792

### CODE REVIEW

File: en/contact-us.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Title: Contact Us
+table_of_contents: True
```
Comment (evilnick) [inline en/contact-us.md]: is this required? there aren't any other headings

File: en/contact-us.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+# Contact us
+
+We love hearing your feedback and about how you're using MAAS. Get in touch
+with the team from one of the following locations:
```
Comment (evilnick) [inline en/contact-us.md]: maybe just 'via one of the following:'

File: en/contact-us.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- On the [MAAS IRC channel][irc-freenode], hosted on [freenode][freenode].
```
Comment (evilnick) [inline en/contact-us.md]: ~~On~~

File: en/contact-us.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- The MAAS [developers mailing list][mailing-list]; a great place to ask
+  questions.
+- Via the [Launchpad][maas-launchpad] source code and issue tracking repository.
```
Comment (evilnick) [inline en/contact-us.md]: ~~Via~~

Comment (degville) [reply en/contact-us.md]: thanks evilnick - I added those because I dislike starting each sentence with 'The'.

Comment (evilnick) [reply en/contact-us.md]: me too, but it sounds a bit weird

**Pull request merged**